package.json

```diff
@@ -35,6 +35,7 @@
     "fs-extra": "4.0.2",
     "inquirer": "1.2.3",
     "is-valid-path": "0.1.1",
+    "preferences": "^0.2.1",
```

Comment:
Remove carrot from preferences

Reply:
Why? Wouldn't that let us pull in patches?

Reply:
@nischayv would have a better answer to this. He had us remove them earlier, I'm just trying to stay consistent.